src/python/nimbusml/internal/utils/entrypoints.py

```diff
@@ -141,6 +141,12 @@ def _get_temp_file(suffix=None):
     return file_name
 
 
+class DataOutputFormat(Enum):
```

Comment:
Rename:
IDV (in docs point to the idv public page for description)
CSR
DF (dataframe)

Use numbers for values.